registration/locale/de/LC_MESSAGES/django.po

```diff
@@ -152,14 +208,100 @@ msgstr "Nein"
 msgid "No contributions yet."
 msgstr "Es sind keine Beiträge vorhanden."
 
-#: licenses/admin.py:39
+#: licenses/admin.py:31 licenses/models.py:66
+msgid "Description"
+msgstr "Beschreibung"
+
+#: licenses/admin.py:33 licenses/models.py:72
+msgid "Further involved persons"
+msgstr "Weitere Personen"
+
+#: licenses/admin.py:35 licenses/models.py:25 licenses/models.py:39
+#: licenses/models.py:154 licenses/templates/licenses/details.html:16
+msgid "Category"
+msgstr "Kategorie"
+
+#: licenses/admin.py:36 licenses/models.py:85
+msgid "Suggested broadcast date"
+msgstr "Gewünschtes Sendedatum"
+
+#: licenses/admin.py:37 licenses/models.py:91
+msgid "Repetitions allowed"
+msgstr "Wiederholungen erlauben"
+
+#: licenses/admin.py:40 licenses/models.py:97
+msgid "Media Authority exchange allowed"
+msgstr "Austausch zwischen Bürgermedien erlauben"
+
+#: licenses/admin.py:43 licenses/models.py:103
+msgid "Youth protection necessary"
+msgstr "Jugendschutz notwendig"
+
+#: licenses/admin.py:45 licenses/models.py:109
+msgid "Store in OK media library"
+msgstr "In OK-Mediathek speichern"
+
+#: licenses/admin.py:46 licenses/models.py:165
+msgid "Screen Board"
+msgstr "Bildschirmtafel"
+
+#: licenses/admin.py:47
+msgid "created at"
+msgstr "erstellt am"
+
+#: licenses/admin.py:59
+msgid "Creation year"
+msgstr "Erstellungsjahr"
+
+#: licenses/admin.py:96
+msgid "<= 10 minutes"
+msgstr "<= 10 Minuten"
+
+#: licenses/admin.py:97
+msgid "<= 30 minutes, > 10 minutes"
+msgstr "<= 30 Minuten, > 10 Minuten"
+
+#: licenses/admin.py:98
+msgid "<= 1 hour, > 30 minutes"
+msgstr "<= 1 Stunde, > 30 Minuten"
+
+#: licenses/admin.py:99
+msgid "> 1 hour "
+msgstr "> 1 Stunde "
```

Comment:
This looks like a smell. If there should be a whitespace needed in the UI, it should be added programmatically and not in the message-id/translation.

Reply:
[5956aa6](https://github.com/gocept/ok_tools/pull/87/commits/5956aa6851a8ca5dfe0273bd27b6c0470bf444a4)